django/core/management/commands/listurls.py

```diff
@@ -0,0 +1,275 @@
+import json
+from importlib import import_module
+from io import StringIO
+
+from django.conf import settings
+from django.contrib.admindocs.views import (
+    extract_views_from_urlpatterns,
+    simplify_regex,
+)
+from django.core.management import color
+from django.core.management.base import BaseCommand, CommandError, CommandParser
+from django.utils import termcolors
+
+FORMATS = (
```

Comment:
You asked about colour; I've seen `--no-color` used as a flag before, maybe use that?

Reply:
It's being used to decide to apply coloring

```python
# Apply colors
if (
    color.supports_color()
    and (not no_color)
    and (format not in COLORLESS_FORMATS)
):
    url_patterns = self.apply_color(url_patterns=url_patterns)
```